src/Sylius/Component/Customer/Model/Customer.php

```diff
@@ -300,16 +268,6 @@ public function setPhoneNumber($phoneNumber)
 
     public function __toString()
     {
-        return (string) $this->getEmail();
-    }
-
-    /**
-     * @param UserInterface $user
-     */
-    protected function assignCustomer(UserInterface $user = null)
-    {
-        if (null !== $user) {
-            $user->setCustomer($this);
-        }
+        return (string)$this->getEmail();
```

Comment:
If email is not nullable, then it isn't necessary.
If email is nullable, then we should fix the CS and make it `(string) $this->getEmail();` :tada: